tools/dependencies/known-dependencies.txt

```diff
@@ -91,18 +76,8 @@ hadoop-yarn-client-3.2.4.jar
 hadoop-yarn-common-3.2.4.jar
 hbase-noop-htrace-4.1.1.jar
 hibernate-validator-6.2.2.Final.jar
-hive-common-2.3.9.jar
-hive-jdbc-2.3.9.jar
-hive-metastore-2.3.9.jar
-hive-serde-2.3.9.jar
-hive-service-2.3.9.jar
-hive-service-rpc-2.3.9.jar
-hive-storage-api-2.4.0.jar
-httpasyncclient-4.1.5.jar
```

Comment:
This lib will not exist on binary package?

Reply:
Yes. All plugins will be uploaded to maven as shade packages.